tests/ui/needless_continue.rs

```diff
@@ -51,6 +51,19 @@ fn main() {
     }
 }
 
+fn simple_loop() {
+    loop {
+        continue; // should lint here
+    }
+}
+
+fn simple_loop2() {
+    loop {
+        println!("bleh");
+        continue; // should lint here
+    }
+}
```

Comment:
Thanks for extending it! With that I just thought of another case: No semicolon after the `continue`. I would add a test for this for both cases, the simple one with only `continue` in the loop and the more general case with something before `continue` in the loop.

Reply:
Covered this case. I am not sure how much value it adds as this code is automatically changed to `continue;` once user runs fmt (but it is more consistent).